node/src/Commands.ts

```diff
@@ -2463,7 +2465,7 @@ export function createBitCount(
 
     if (options) {
         args.push(options.start.toString());
-        args.push(options.end.toString());
+        if (options.end) args.push(options.end.toString());
```

Comment:
```suggestion
        if (options.end !== undefined) args.push(options.end.toString());
```
condition `if (options.end)` will be `false` if `end` is 0

Reply:
good catch